src/components/registration-view/registration-view.jsx

```diff
@@ -2,9 +2,13 @@ import react from 'react';
 import axios from 'axios';
 import { Button } from 'react-bootstrap';
 import { useState } from 'react';
+import { Form } from 'react-bootstrap';
+
 export function RegistrationView(props) {
 const [ username, setUsername ] = useState('');
 const [ password, setPassword ] = useState('');
+const [email, setEmail] = useState('');
+const [birthday, setbirthday] = useState('');
```

Comment:
It seems like this Component is still a work in progress but the logic should be pretty similar to the LoginView. You just want to make sure that you're passing all of the required arguments when you send the request to the API